.gitmodules

```diff
@@ -74,7 +74,7 @@
 	branch = px4
 [submodule "Tools/simulation/gz"]
 	path = Tools/simulation/gz
-	url = https://github.com/PX4/PX4-gazebo-models.git
+	url = git@github.com:OpenHORYZN/PX4-gazebo-models.git
```

Comment:
This should not be changed. If you need changes from your side, you need to merge them in PX4/PX4-gazebo-models first. Please open a pull-request there.